icons/ufo.svg

```diff
@@ -0,0 +1,19 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M18 8c0 1-3 2-6 2S6 9 6 8a6 6 0 0 1 12 0" />
+  <path d="M7 13h.01" />
+  <path d="M12 14h.01" />
+  <path d="M17 13h.01" />
+  <path d="M6 8.1c-2.4 1-4 2.6-4 4.4 0 3 4.5 5.5 10 5.5s10-2.5 10-5.5c0-1.8-1.6-3.4-4-4.4" />
+  <path d="m7 22 2-4" />
+  <path d="m17 22-2-4" />
```

Comment:
The "feet" look a bit weird, and yet again, can you provide use cases for this one? 🤔